packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3616,6 +3616,21 @@ class MergeSemantics extends SingleChildRenderObjectWidget {
   RenderMergeSemantics createRenderObject(BuildContext context) => new RenderMergeSemantics();
 }
 
+/// A widget that drops the semantics of all widget that were painted before it
+/// in the same semantic container.
+///
```

Comment:
please add some more depth to this specific documentation, e.g. giving an example of where you'd use this